contracts/deploy/018_upgrade_governor.js

```diff
@@ -0,0 +1,235 @@
+      signature: "claimGovernance()",
+    },
+  ]);
+
+  if (isMainnet) {
+    // On Mainnet, only propose. The enqueue and execution are handled manually via multi-sig.
+    log("Sending transfer proposal to old governor...");
+    await sendProposal(propTransferArgs, propTransferDescription, {
+      governorAddr,
+    });
+    log("Transfer proposal sent.");
+
+    log("Sending claim proposal to new governor...");
+    await sendProposal(propClaimArgs, propClaimDescription);
+    log("Claim proposal sent.");
+  } else if (isFork) {
+    // On Fork we can send the proposal then impersonate the guardian to execute it.
+    log("Sending and executing transfer proposal...");
+    // Note: we send the proposal to the old governor by passing explicitly its address.
+    await executeProposal(propTransferArgs, propTransferDescription, {
+      governorAddr,
+    });
+
+    log("Sending and executing claim proposal...");
+    await executeProposal(propClaimArgs, propClaimDescription, {
+      guardianAddr: addresses.mainnet.Guardian,
+    });
+  } else {
+    // Hardcoding gas estimate on Rinkeby since it fails for an undetermined reason...
+    const gasLimit = isRinkeby ? 1000000 : null;
+    await withConfirmation(
+      cOUSDProxy
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on OUSDPoxy");
+
+    await withConfirmation(
+      cVaultProxy
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on VaultProxy");
+
+    await withConfirmation(
+      cCompoundStrategyProxy
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on CompoundStrategyProxy");
+
+    await withConfirmation(
+      cThreePoolStrategyProxy
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on ThreePoolStrategyProxy");
+
+    await withConfirmation(
+      cAaveStrategyProxy
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on AaveStrategyProxy");
+
+    await withConfirmation(
+      cBuyback
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on BuyBack");
+
+    await withConfirmation(
+      cOGNStakingProxy
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on OGNStakingProxy");
+
+    await withConfirmation(
+      cCompensationClaim
+        .connect(sGovernor)
+        .transferGovernance(dGovernor.address, await getTxOpts(gasLimit))
+    );
+    log("Called transferGovernance on CompensationClaim");
```

Comment:
This doesn't seem to test the claim governance portion of the changes on Rinkby?

Reply:
It does not. Looking more into it, this won't even work on Rinkeby since the owner is not a EOA but a timelock governor contract... I'll just remove it. Fork validation should be sufficient.